assets/icons/chatbot-person.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="400" height="400" viewBox="0 0 400 400" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
:wrench: I think we decided on the naming `chat`, not `chatbot`, could you please rename the two icons?